config/locales/en.yml

```diff
@@ -178,7 +178,10 @@ en:
       no_local_authority_url_html: We don't have a link for their website. Try the <a href="/find-local-council">local council search</a> instead.
       local_authority_website: Go to %{local_authority_name} website
       no_website: We don't have a link for their website.
-      info_on_country_website: Get information on this service, or a similar service, on the %{country_name} Government website.
+      info_on_country_website:
+        northern_ireland: Get information on this service, or a similar service, on the Northern Ireland Executive website.
+        scotland: Get information on this service, or a similar service, on the Scottish Government website.
+        wales: Get information on this service, or a similar service, on the Welsh Government website.
```

Comment:
The duplication here seems a bit unnecessary, I wonder if it can be removed 🤔 

Maybe something like...

```
government_name:
  northern_ireland: Northern Ireland Executive
  scotland: Scottish Government
  wales: Welsh Government
info_on_country_website: Get information on this service, or a similar service, on the %{government_name} website.
```

Then call it...

``` 
<%= t("formats.local_transaction.info_on_country_website", government_name: t("formats.local_transaction.government_name.#{@country_name.parameterize(separator: "_")}")) %>
```

Reply:
As there are only 3 countries, I think it's fine to update content 3 times in the locale file. If there were over 10, then yes, we'd have to think of a nicer way to generate the content. I think that accessing two locale entries in the view like that will make it harder for future developers to understand what is going on as well.